microsoft-edge/devtools-guide-chromium/whats-new/2023/03/devtools-111.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: What's new in DevTools (Microsoft Edge 111)
```

Comment:
```suggestion
title: What's new in DevTools (Microsoft Edge 112)
```
Should be 112, instead of 111.

Reply:
@michaelfarnsworth do you mean that this h2 section/entry should be moved from What's New 111 to What's New 112?
* https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2023/04/devtools-112
* https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2023/03/devtools-111
   * https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2023/03/devtools-111#remote-debugging-of-microsoft-edge-pwas-and-webview2-controls-on-xbox-and-hololens-devices

Reply:
fyi for awareness, note that the HoloLens GetStart PR adds "2" after HoloLens in its [What's New 111](https://github.com/MicrosoftDocs/edge-developer/pull/2567/files#diff-b1a613b570630c83b92b81d400898b19b594953b198e37f7a41b4c2fd7a834a3)

Reply:
Good question. Yeah, we shouldn't change the title of this file. I was passing on some feedback second-hand. I think this is probably fine, and that we don't need to move anything to 112. I'll double-check.

Reply:
Disregard.